ports/opencv/vcpkg.json

```
@@ -26,6 +27,12 @@
         }
       ]
     },
+    "aravis": {
+      "description": "aravis",
+      "dependencies": [
+        "aravis"
```

This port is just forwarding to the implementation.
```suggestion
        {
          "name": "opencv4",
          "default-features": false,
          "features": [
            "aravis"
          ]
        }
```

---

good catch! that was a copy&paste mistake 🤦